datahub-graphql-core/src/main/resources/app.graphql

```diff
@@ -66,6 +66,11 @@ type PlatformPrivileges {
   Whether the user is able to manage UI-based secrets
   """
   manageSecrets: Boolean!
+
+  """
+  Whether the user should be able to manage tokens on behalf of other users.
```

Comment:
This works - but I'd rather change the privilege to be "MANAGE-TOKENS" instead of what it is now